index.js

```diff
@@ -0,0 +1,96 @@
+let firstNumber;
+let display = document.querySelector('.display')
+display.textContent="";
+
+let clear = document.querySelector('#clear')
+let operators = document.querySelectorAll('.operator')
+let operator;
+let results;
+let secondNumber;
+let getElement = document.querySelectorAll('#click')
+let operatorClicked = false;
+
+const sum = ()=>{
+  results = Number(firstNumber + secondNumber); 
+  return firstNumber + secondNumber;
+  
+}
+const substract = ()=>{
+
+  results =  Number(firstNumber - secondNumber);
+  return firstNumber - secondNumber;
+
+}
+const divide = ()=>{
+  results = Number(firstNumber / secondNumber);
+  return firstNumber / secondNumber;
+}
+const multiply = ()=>{
+  results = Number(firstNumber * secondNumber);
+  return firstNumber * secondNumber;
+}
+const operate = () => {
+  if (firstNumber !== undefined && secondNumber !== undefined) {
+    if (operator == '+') {
+      results = sum();
+    } else if (operator == '-') {
+      results = substract();
+    } else if (operator == '/') {
+      results = divide();
+    } else if (operator == '*') {
+      results = multiply();
+    }
+    firstNumber = results;
+    display.textContent = results;
```

Comment:
Here you don't respect the so called "single responsibility" concept. This means that a class / function / method / component should always have a specific goal and don't do anything else.

A method called `operate` should exactly do that -> operate the result. But better keep out the displaying of the result from here and move it one level up to the event handlers (I think you anyway have the displaying logic there as well, so you probably just have to drop it here). Why single responsibility? Because for better reuse of functionality.